src/selectors/daoSelectors.ts

```diff
@@ -19,26 +20,31 @@ const getDaoProposals = createSelector(
   },
 );
 
-export const makeDaoBoostedProposalsSelector = () => {
-  return createSelector(
-    [ getDaoProposals ],
-    (proposals) => proposals.filter((proposal: IProposalState) => (proposal.state == ProposalStates.Boosted)),
-  );
-};
-
-export const makeDaoPreBoostedProposalsSelector = () => {
-  return createSelector(
-    [ getDaoProposals ],
-    (proposals) => proposals.filter((proposal: IProposalState) => (proposal.state == ProposalStates.PreBoosted)),
-  );
-};
-
-export const makeDaoExecutedProposalsSelector = () => {
-  return createSelector(
-    [ getDaoProposals ],
-    (proposals) => proposals.filter((proposal: IProposalState) => (proposal.state == ProposalStates.Executed || proposal.state == ProposalStates.Closed)),
-  );
-};
+export const boostedProposals = createSelector(
```

Comment:
I used functions because if there are multiple components using the same selector with different proposals (multiple DAOs for example) then it won't memoize correctly: https://github.com/reduxjs/reselect I think we still need to do that to future proof

Reply:
Correct. Another problem I noticed is that, since the selectors are time-dependent there's no point in memoization anyway. 

Do we actually need this performance benefit in the app? seems to me that it's negligible (unless there are thousands of proposals, in which case I think we should display and load them in pages, i.e. infinite scroll or similar)

What do you think?

Reply:
I think we get some benefit from memoization, unless we expect there to be a constant flood of new proposals all the time. And in the future there will likely be multiple components using these selectors, which will make them more valuable. At the very least we need similar functions anyway to filter the proposals, so why not make them selectors :)

Reply:
But since it's time-dependent, memoization can lead to stale views, that should be updated (because time passes), but are not. I guess that's not such a big concern.

Reply:
fixed: https://github.com/daostack/alchemy/pull/74/commits/723b55a98ed4cfda68596736869992856a104b5a

Reply:
the views should never be stale, reselect invalidates the memoization if the proposal list changes at all